.eslintrc.json

```diff
@@ -103,7 +103,6 @@
     "no-octal-escape": 2,
     "no-path-concat": 0,
     "no-proto": 2,
-    "no-redeclare": 2,
```

Comment:
I had to remove this to make TypeScript function overloading work. We could enable the TypeScript version if we want, but I didn't see the point. I think TS will warn you anyway.

Reply:
Yes, please. eslint is stupid.